documents/model.go

```diff
@@ -290,21 +301,23 @@ func (m *CoreDocumentModel) GetDocumentTree() (tree *proofs.DocumentTree, err er
 // GetDocumentSigningTree returns the merkle tree for the signing root
 func (m *CoreDocumentModel) GetDocumentSigningTree(dataRoot []byte) (tree *proofs.DocumentTree, err error) {
 	// coredoc tree
-	coreDocTree, err := m.GetDocumentTree()
+	coreDocTree, err := m.GetCoreDocumentTree()
 	if err != nil {
 		return nil, err
 	}
 
 	// create the signing tree with data root and coredoc root as siblings
-	tree = NewDefaultTree(ConvertToProofSalts(m.Document.CoredocumentSalts))
+	tree = NewDefaultTreeWithPrefix(ConvertToProofSalts(m.Document.CoredocumentSalts), SigningTreePrefix, compactProperties(SigningTreePrefix))
+	parentProp := NewLeafProperty(SigningTreePrefix, compactProperties(SigningTreePrefix))
```

Comment:
Why not call it propPrefix?

```suggestion
	propPrefix := NewLeafProperty(SigningTreePrefix, compactProperties(SigningTreePrefix))
```